deidentification/info.py

```diff
@@ -0,0 +1,22 @@
+version_major = 1
+version_minor = 0
+version_micro = 0
+version_extra = ''
+
+__version__ = '{}.{}.{}{}'.format(version_major,
```

Comment:
Should we use f-string now ?

Reply:
As python version for deidentification must be >= 3.6, we could use f-string whitout issues.